File: fromJAVAtoPYTHON/04_OOP/05.Exercise_Part_2_Create_a_new_Class/Bike.java
public class Bike {
    int gear;
    int speed;

    Bike(int gear, int speed) {
        this.gear = gear;
        this.speed = speed;
    }

    public String toString() {
        return "(" + gear + ", " + speed + ")";
    }

    public static void main(String[] args) {
        Bike honda = new Bike(3, 200);
        Bike ducati = new Bike(1, 15);

        System.out.println(honda);
        System.out.println(ducati);
    }
}

File: fromJAVAtoPYTHON/04_OOP/05.Exercise_Part_2_Create_a_new_Class/Bike.py
class Bike:
    # constructor
    def __init__(self, gear, speed) -> None:
        self.gear = gear
        self.speed = speed

    #toString
    def __repr__(self) -> str:
        return repr((self.gear, self.speed))

#instances
honda = Bike(3, 50)
ducati = Bike(1, 10)

print(honda)
print(ducati)